.github/workflows/macos.yml

```diff
@@ -1,64 +1,51 @@
-name: Build for MacOS
+name: Build for macOS
```

Comment:
Is this the official writing? With small "m"?

Reply:
Yes, they spells it as "macOS".